hypha.php

```diff
@@ -293,10 +293,8 @@ function install() {
 			if (!is_writable('.')) $errorMessage.= 'php has no write access to the hypa installation directory on the server<br/>';
 
 			// extract list of languages
-			$iso639 = data('system/core/language.php');
-			$p1 = strpos($iso639, "json_decode('");
-			$p2 = strpos($iso639, "', true);", $p1);
-			$iso639 = json_decode(preg_replace('/\\\\\'/', '\'', substr($iso639, $p1+13, $p2-$p1-13)));
+			$iso639 = data('system/languages/languages.json');
+			$iso639 = json_decode(preg_replace('/\\\\\'/', '\'', $iso639));
```

Comment:
Why do we still need this replacement? Isn't the content of `languages.json` valid json as-is?

Reply:
Ah, it seems these `\'` is present in the `languages.json` file you added. These were probably needed when the json was contained in a PHP `''` string, but this is no longer relevant. I would rather remove these backslashes from the `languages.json` file and remove the replacement above.

Reply:
This is resolved.